libtelemetry/src/main/java/com/mapbox/android/telemetry/navigationutils/AudioTypeChain.java

```diff
@@ -1,4 +1,4 @@
-package com.mapbox.android.telemetry;
+package com.mapbox.android.telemetry.navigationutils;
```

Comment:
Could we use the `navigation.utils` package structure as suggested in https://github.com/mapbox/mapbox-events-android/pull/67#pullrequestreview-98119629?